yw_website/yw_website/apps/website/templates/pages/run_detail.html

```diff
@@ -8,6 +8,9 @@
 
 <div class="box fit-page">
 <div class="border">
+    <div class="edit_button">
+            <button onclick="window.history.back();" class="waves-effect waves-light btn" > Back</button>
```

Comment:
I think this should instead redirect to the associated workflows url, in case someone navigated to it differently. Right now this serves no separate purpose to the browser back button

Reply:
Bowers suggested we make the `workflow` section clickable instead of a back button

> I think this should instead redirect to the associated workflows url

agreed

Reply:
You can just surround line 18/21 (depending on what version of the file you're looking at) with an `<a>` tag likes so:

```
<a href={% url 'detailed_workflow' run.version.id %}><b>Workflow: </b> {{run.version.workflow.title}} </a>
```